Orange/classification/random_forest.py

```diff
@@ -2,7 +2,7 @@
 
 from Orange.classification import SklLearner, SklModel
 
-__all__ = ["RandomForestLearner"]
+__all__ = ["RandomForestLearner", "RandomForestClassifier"]
```

Comment:
It was decided that __all__ should contain only classes that a user should see (and would use).
If other classes are needed elsewhere in the code they should be imported explicitly with the full path.